CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### v10.1.0 (2024-03-04)
+
+* Migrated to using prebuildify and node-gyp-install.

```

Comment:
A few things I would call out:
 * prebuilts are now shipped with package
 * call out which prebuilts exist now
 * call out the removal of the env vars `NR_NATIVE_METRICS_NO_DOWNLOAD` `NR_NATIVE_METRICS_NO_BUILD`, `NR_NATIVE_METRICS_DOWNLOAD_HOST`, and `NR_NATIVE_METRICS_REMOTE_PATH`